packages/gcp-cloud-run/src/executors/schema.d.ts

```diff
@@ -23,7 +23,6 @@ export interface ExecutorSchema {
   buildWith?: 'artifact-registry'
```

Comment:
Do you think we'll ever expand the option for this? Not sure if you ever follow a standard for literal types like this, but we could also move this to a separate type as `type buildWithOpts = 'artifact-registry'`, which would then be more easily extendible. We could then also move the ingress options to a similar type. ( this could potentially also clean up the ingressOpts check in the runner + the `===` checks for buildWith` ).

Reply:
Currently I just mapped everything on how it was with the SDK, think we can indeed improve here.

If you want you could do a small write up to explain a bit more how you would like to see this.

Reply:
> Currently I just mapped everything on how it was with the SDK, think we can indeed improve here.
> 
> If you want you could do a small write up to explain a bit more how you would like to see this.

I can do that in a follow-up PR, let's leave this PR just for documentation. If you agree with the changes so far let's merge.